cmd/xl-v1-object.go

```diff
@@ -643,12 +690,12 @@ func (xl xlObjects) PutObject(bucket string, object string, size int64, data io.
 		IsDir:           false,
 		Bucket:          bucket,
 		Name:            object,
-		Size:            xlMeta.Stat.Size,
-		ModTime:         xlMeta.Stat.ModTime,
-		MD5Sum:          xlMeta.Meta["md5Sum"],
-		ContentType:     xlMeta.Meta["content-type"],
-		ContentEncoding: xlMeta.Meta["content-encoding"],
-		UserDefined:     xlMeta.Meta,
+		Size:            partsMetadata[0].Stat.Size,
```

Comment:
Indicate why we are using first element in the 'partsMetadata' slice.  Rather than changing this just set 

```
xlMeta := partsMetadata[0]
```